README.md

```diff
@@ -6,6 +6,8 @@
 
 Create React components in a way analogous to `React.createClass`, but powered by a [subset](docs/api.md#api-differences) of the [stampit](https://github.com/stampit-org/stampit) API.
 
+**This library is no longer maintained and is being replaced by the stamp spec compliant [react-stamp](https://github.com/troutowicz/react-stamp).**
```

Comment:
Probably not, but do we want to send the message, "This repo is obsolete, and the new one is not ready and unstable"?